packages/runtime/client/rest/src/withdrawal/endpoints/singleton.ts

```diff
@@ -11,16 +11,26 @@ import { formatValidationErrors } from "jsonbigint-io-ts-reporters";
 import { WithdrawalDetails } from "../details.js";
 import { DecodingError } from "@marlowe.io/adapter/codec";
 import {
+  BlockHeader,
   HexTransactionWitnessSet,
   WithdrawalId,
   transactionWitnessSetTextEnvelope,
   unWithdrawalId,
 } from "@marlowe.io/runtime-core";
+import { TxStatus } from "../../contract/transaction/status.js";
+import { PayoutHeader } from "../../payout/header.js";
 
 export type GET = (
   withdrawalId: WithdrawalId
 ) => TE.TaskEither<Error | DecodingError, WithdrawalDetails>;
 
+export type GetWithdrawalByIdResponse = {
+  withdrawalId: WithdrawalId;
+  status: TxStatus;
+  block: BlockHeader | null;
```

Comment:
we normally use `block?: BlockHeader` instead. I'm merging this now as it is a minor comment, but could you address this later?

Reply:
Good point. Fixed.